packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -780,13 +864,19 @@ contract Attestations is
       currentIndex++;
       attestations.status = AttestationStatus.Incomplete;
       // solhint-disable-next-line not-rely-on-time
-      attestations.time = uint128(now);
+      attestations.blockNumber = uint64(block.number);
       state.issuers.push(issuer);
     }
   }
 
-  function isAttestationTimeValid(uint128 attestationTime) internal view returns (bool) {
+  function isAttestationRequestBlockTimeValid(
+    uint128 attestationRequestBlockTime
+  )
+    internal
+    view
+    returns (bool)
+  {
     // solhint-disable-next-line not-rely-on-time
-    return now < attestationTime.add(attestationExpirySeconds);
+    return block.number < uint256(attestationRequestBlockTime).add(attestationExpiryBlocks);
```

Comment:
Why are we using time and block numbers? Seems like we should pick one

Reply:
It's blocks all throughout, somehow I thought using `blockTime` would indicate this denotes a time in block units, vs. a duration in block units, but see how this is more confusing. Changed it to `attestationRequestBlock`